calico/felix/fetcd.py

```diff
@@ -284,11 +274,11 @@ def watch_etcd(self, update_splitter):
                     _log.warning("Unexpected action %s to %s; triggering "
                                  "resync.", response.action, response.key)
                     continue_polling = False
-                if response.key.startswith("/calico/config"):
+                if response.key.startswith(CONFIG_DIR):
                     _log.warning("Global config changed but we don't "
-                                 "yet support dynamic config response: %s",
-                                 response)
-                if response.key.startswith(self.my_config_prefix):
+                                 "yet support dynamic config key=%s",
+                                 response.key)
```

Comment:
I think that might be a merge conflict resolution mistake.  I changed it to response so that it dumps the whole response including its key, action and data.  response is correct.